site/content/blog/2022-08-30-spier-move/index.md

```diff
@@ -0,0 +1,51 @@
+---
+title: "Spider News"
+summary: >
+  News about changes to the Traditional Spider for the up-coming release.
+images:
+- https://www.zaproxy.org/blog/2022-08-30-spider-move/images/zapbot.spider.jpg
+type: post
+date: "2022-08-30"
+authors:
+    - thorin
+---
```

Comment:
Oops I forgot to include the social preview front matter. Will add it in the morning.